### DISCUSSION TITLE
第2节课笔记(19班)

### DISCUSSION THREAD

**Discussion by LYMDLUT** [InternLM/Tutorial#106] [笔记]
第2节课笔记(19班)

**Comment by Sixdes** [comment]
https://bbs.csdn.net/topics/617868650?spm=1001.2014.3001.6377

**Comment by Jiangzzzzl** [comment]
https://blog.csdn.net/2301_80618119/article/details/135431085?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135431085%22%2C%22source%22%3A%222301_80618119%22%7D

**Comment by jts250** [comment]
https://zhuanlan.zhihu.com/p/676483224

**Comment by hu-qi** [comment]
https://zhuanlan.zhihu.com/p/676512153

**Comment by romanwin28** [comment]
https://blog.csdn.net/George_han/article/details/135443445

**Comment by jia-heng** [comment]
https://github.com/jia-heng/puyu-learn/blob/main/lesson2.md

**Comment by AllYoung** [comment]
https://blog.csdn.net/sunshine_youngforyou/article/details/135439284

**Comment by melonwine** [comment]
https://github.com/melonwine/InternLMPlay/blob/master/02/note02.md

**Comment by mkdirmushroom** [comment]
[第2节课笔记](https://github.com/mkdirmushroom/note/blob/main/InternLM/tutorial1_note.md)

**Comment by Hamasuta666** [comment]
第2节课笔记（作业在文档末尾）：
https://zhuanlan.zhihu.com/p/680561771
